libraries/SPI/src/utility/xmc_spi_conf.c

```diff
@@ -107,46 +107,6 @@ XMC_SPI_t XMC_SPI_0 =
     }
 };
 
-#elif defined(XMC1400_Arduino_Kit) 
```

Comment:
Mmm ... the fork has not been updated to the latest develop-2.x branch. And the board is removed from the built-in library.  I believe this is one of the boards to keep in 2.x.

Reply:
Yes it should stay as the XMC1400 will be the successor or the XMC1100

Reply:
Yes. Sorry. I missed out on the updates inside the libraries folder. Will update that